### PR TITLE
setDeepLinkConfig

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: packages/wallet-sdk/src/provider/MobileRelayUI.ts [hunk 1]
```diff
@@ -41,12 +41,24 @@ export class MobileRelayUI implements WalletUI {
...
     this.useLocationMethod = useLocationMethod;
+    this.useCustomScheme = useCustomScheme;
   }
 
   private redirectToCoinbaseWallet(walletLinkUrl?: string): void {
-    const url = new URL("https://go.cb-w.com/walletlink");
+    const baseUrl = this.useCustomScheme
+      ? "cbwallet://"
+      : "https://go.cb-w.com/";
+    const url = new URL("walletlink", baseUrl);
```
Comment (vishnumad) [inline packages/wallet-sdk/src/provider/MobileRelayUI.ts:61]: The url opened seems to be `cbwallet:///walletlink?redirect_url...`. Note the three `///`

**Comment by vishnumad** [comment]
I'm seeing some weird behavior when testing this change in a WKWebView. I updated your branch locally to use valid `cbwallet://walletlink` urls, but it seems like if I cancel the initial request and come back and send another request, it users the `https://go.cb-w.com` url again. 🤔 

image

File: packages/wallet-sdk/src/provider/MobileRelayUI.ts (hunk 1 above)
Comment (bangtoven) [reply packages/wallet-sdk/src/provider/MobileRelayUI.ts:61]: thanks. what would be the optimal url for cbwallet walletlink path?
would it make the logic simpler if we have it the same format like `cbwallet://go.cb-w.com/walletlink`

**Comment by bangtoven** [comment]
not gonna have this patch

**Pull request closed without merging**